src/test/java/org/jboss/xavier/integrations/route/XmlRoutes_RestReportTest.java

```diff
@@ -63,38 +63,62 @@ public void setup() {
     }
 
     @Test
-    public void xmlRouteBuilder_RestReport_SummaryParamGiven_ShouldCallFindReportSummary() throws Exception {
+    public void xmlRouteBuilder_RestReport_NoParamGiven_ShouldCallFindReports() throws Exception {
         //Given
         camelContext.setTracing(true);
         camelContext.setAutoStartup(false);
 
         //When
         camelContext.start();
         camelContext.startRoute("reports-get-all");
-        Map<String, String> variables = new HashMap<>();
-        variables.put("summary", "true");
-        restTemplate.getForEntity(camel_context + "report?summary={summary}", String.class, variables);
+        restTemplate.getForEntity(camel_context + "report", String.class);
 
         //Then
-        verify(analysisService).findReports(0, 10);
+        verify(analysisService).findReports(anyInt(), anyInt());
```

Comment:
I think we should check that default values (`page="0"` and `size="10"`) for the params are applied in this test

Reply:
My last commit changed anyInt() by static pagination values